docs/source/api.rst

```diff
@@ -25,21 +25,12 @@ There's also a HAWC client available:
 
```

Comment:
can you also add a new subsection in `developer.rst` that documents the command on how to build the client?

it should be something like:

```bash
python client/setup.py sdist bdist_wheel
```